hypothesis-python/RELEASE.rst

```diff
@@ -0,0 +1,4 @@
+RELEASE_TYPE: minor
+
+This release allows using ``Annotated[]`` and ``ReadOnly[]`` types
+for ``TypedDict`` values.
```

Comment:
Let's use sphinx cross-references for these names, and explicitly mention that it's _in `from_type()`_.

Reply:
Sure, will do soon!